### ISSUE TITLE
removal of hacktoberfest tag for update.md 

### ISSUE THREAD

**Issue by prajna-spec** [wbhoomika/python4beginners-Hacktoberfest21#204]
Hello!
I believe u would have to remove the hacktoberfest tag for update.md pull requests because that doesn't count it being a valuable
pull request for hacktoberfest

**Comment by wbhoomika** [comment]
Hi @prajna-spec, 
Apologies but I can't seem to find updated.md file. Are you talking about "About" section? It would be great if you could direct me to the location of the file.

**Comment by prajna-spec** [comment]
![Screenshot 2021-11-03 131054](https://user-images.githubusercontent.com/64818027/140024006-ddba3126-4c64-4693-afec-9bcf973afff3.png)

What I meant is these ones. I don't think they are relevant

**Comment by wbhoomika** [comment]
Hi, that was done under Hacktober fest thus it has that lable. All the requests coming since 1st Nov I'm not adding that lable.

**Comment by wbhoomika** [comment]
And I got your point as it's not a contribution to add name in contributing.md file. But still it was done under Hacktober fest.

**Comment by prajna-spec** [comment]
I agree. but what i'm trying to say is adding a name in update.md is being counted as an additional pr. 
something like this
![Screenshot 2021-11-05 114329](https://user-images.githubusercontent.com/64818027/140466657-1543eb75-f6cd-4686-ae2a-b7dc66a8b626.png)
I'm not entirely sure if this is accepted by hacktoberfest. Could u pls check with someone

**Comment by wbhoomika** [comment]
Sure will do
